test/integration/user-management-api/service_contracts_controller_test.rb

```diff
@@ -7,10 +7,15 @@ def setup
     @service_plan     = FactoryBot.create(:service_plan, issuer: service)
     @application_plan = FactoryBot.create(:application_plan, issuer: service)
 
+    new_service = FactoryBot.create(:simple_service, account: current_account)
+    @new_service_plan     = FactoryBot.create(:service_plan, issuer: new_service)
+    @new_application_plan = FactoryBot.create(:application_plan, issuer: new_service)
+
     @buyer            = FactoryBot.create(:buyer_account, provider_account: current_account)
     @service_contract = FactoryBot.create(:simple_service_contract, plan: @service_plan, user_account: @buyer)
 
     @buyer.buy! @application_plan
+    @buyer.buy! @new_application_plan
```

Comment:
```suggestion
```

This step is not needed, and in fact it is very confusing. In real life, a buyer would never have an application in a service when they don't have a service subscription.